rubicon/java/types.py

```diff
@@ -19,7 +19,7 @@
 jbyte = c_byte
 jchar = c_wchar
 jshort = c_short
-jint = c_int
+jint = c_uint32
```

Comment:
Are you sure about this? Aren't Java ints 32 bit signed?

Reply:
Right-o -- I meant to specify c_int32. Fixed now.

I am a little weirded-out to say `c_int` rather than `c_int32`, so I like the latter, but I'm OK if we don't make that change.